python/ray/tests/test_placement_group.py

```diff
@@ -370,6 +370,38 @@ def f():
         ray.get(task_ref)
 
 
+@pytest.mark.parametrize(
+    "ray_start_cluster_head", [{
+        "num_cpus": 2
+    }], indirect=True)
+def test_remove_placement_group2(ray_start_cluster_head):
```

Comment:
This test is not crashing even before this PR. I guess the underline worker crashing doesn't affect the driver from running successfully. Any suggestion for the change?

Reply:
One possible way of testing this is to remove https://github.com/ray-project/ray/blob/master/python/ray/tune/tests/test_tune_restore.py#L209-L216